specs/utilities/distance-utils.tests.js

```diff
@@ -15,6 +15,20 @@ describe('distance-utils', function () {
             chai.expect(() => getDistanceFromLatLonInKm(lat, lon, lat, lon)).to.throw();
         });
 
+        it('throws an error when given a latitude that is a string', function () {
```

Comment:
maybe rename this to something like "throws an error when give a latitude that is a non-enumerable string"?